### PR TITLE
#390: Added fake/real switch

### PR DESCRIPTION
**Pull request by ahmed-shehata**
#390: Added fake/real switch for bug bashing

### CODE REVIEW

File: AtlasUI/AtlasUI/CheckoutSummaryActionsHandler.swift [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import Foundation
 import AtlasSDK
 
-public var AtlasCheckoutUseRealOrderProcess = false
+public var atlasCheckoutUseRealOrderProcess = false
```
Comment (bonkey) [inline AtlasUI/AtlasUI/CheckoutSummaryActionsHandler.swift]: Nah, I feel like swiftlint is wrong here. Better turn off the warning.

File: AtlasDemo/AtlasDemo/ProfileViewController.swift [hunk 2]
```diff
@@ -35,6 +34,13 @@ class ProfileViewController: UIViewController {
             }
         }
 
+        switch atlasCheckoutUseRealOrderProcess {
+        case false:
+            fakeOrderSwitch.selectedSegmentIndex = 0
```
Comment (bonkey) [inline AtlasDemo/AtlasDemo/ProfileViewController.swift]: I feel awkward thrill when somebody if or case simple true/false.
Rather `fakeOrderSwitch.selectedSegmentIndex = atlasCheckoutUseRealOrderProcess ? 1 : 0`

File: AtlasDemo/AtlasDemo/ProfileViewController.swift [hunk 3]
```diff
@@ -74,8 +80,16 @@ class ProfileViewController: UIViewController {
             AppSetup.change(interfaceLanguage: "en")
         }
     }
-    @IBAction func liveServerSwitchChanged(serverSwitch: UISwitch) {
-        AppSetup.change(environmentToSandbox: !serverSwitch.on)
+
+    @IBAction func fakeOrderSwitched(sender: UISegmentedControl) {
+        switch sender.selectedSegmentIndex {
+        case 1:
+            atlasCheckoutUseRealOrderProcess = true
```
Comment (bonkey) [inline AtlasDemo/AtlasDemo/ProfileViewController.swift]: When it's just true/false better `atlasCheckoutUseRealOrderProcess = sender.selectedSegmentIndex == 1`

File: AtlasDemo/AtlasDemo/ProfileViewController.swift [hunk 4]
```diff
@@ -88,4 +102,14 @@ class ProfileViewController: UIViewController {
         return ["en", "de"].indexOf(languageCode) ?? 0
     }
 
+    override func canBecomeFirstResponder() -> Bool {
+        return true
+    }
+
+    override func motionEnded(motion: UIEventSubtype, withEvent event: UIEvent?) {
+        if motion == .MotionShake {
+            self.fakeOrderSwitch.enabled = true
```
Comment (bonkey) [inline AtlasDemo/AtlasDemo/ProfileViewController.swift]: I thought about switching between them with shake, but it's fine this way too.

Comment (ahmed-shehata) [inline AtlasDemo/AtlasDemo/ProfileViewController.swift]: I would also add the 3-knock/shake 😉

File: AtlasDemo/AtlasDemo/ProfileViewController.swift (hunk 2 above)
Comment (ahmed-shehata) [inline AtlasDemo/AtlasDemo/ProfileViewController.swift]: done

**Pull request merged**